src/pages/Cart/Cart.jsx

```diff
@@ -1,13 +1,91 @@
 import React from 'react'
-import { NavBar } from '../HomePage/HomePageComponents/NavBar/NavBar'
+import { NavBar } from '../HomePage/HomePageComponents/NavBar/NavBar';
+import {Footer} from '../HomePage/HomePageComponents/Footer/Footer';
```

Comment:
Here you can make a file index.js in HomePageComponents, where you keep all the exports and then you can import different component in one line like,
`import {NavBar, Footer} from '../HomePage/`

Reply:
I will update that. Thanks for the suggestion